tracing-appender/src/rolling.rs

```diff
@@ -403,24 +403,7 @@ impl FilenameTemplate for Prefixed {
 /// Create a new [`FilenameTemplate`] which uses a common file name and
 /// extension.
 ///
-/// # Examples
-///
-/// ```rust
-/// # use chrono::prelude::*;
-/// # use tracing_appender::rolling::{Rotation, FilenameTemplate};
-/// # use std::path::Path;
-/// let mut template = tracing_appender::rolling::with_name_and_extension(
-///     "/var/log",
-///     "MyApplication",
-///     "log",
-/// );
-///
-/// let date = Utc.ymd(2020, 2, 1).and_hms(12, 20, 15);
-/// let rotation = Rotation::DAILY;
-///
-/// let filename = template.next_log_file(&date, &rotation);
-/// assert_eq!(filename, Path::new("/var/log/MyApplication.2020-02-01.log"));
-/// ```
+/// This will generate a filename like `/var/log/MyApplication.2020-02-01.log`.
```

Comment:
nit: I think we should still show an example here so that people know what part of the filename is the extension and that the extension doesn't include the date.

So like before just show something like 

```rust	
/// ```rust
/// # use chrono::prelude::*;	
/// # use tracing_appender::rolling::{Rotation, FilenameTemplate};	
/// # use std::path::Path;	
/// let mut template = tracing_appender::rolling::with_name_and_extension(	
///     "/var/log",	
///     "MyApplication",	
///     "log",	
/// );
/// ```
```

Reply:
Should I revert the change then? That way you've got a line showing how the `with_name_and_extension()` works, and it also has an `assert!()` at the bottom so you know exactly what filename it generates.

Reply:
I don't believe we generally put assert! statements or test like code in the docs.

See https://docs.rs/tracing-appender/0.1.1/tracing_appender/rolling/index.html for an example. 

It's good enough just to show what filename is generated like in the above examples or in your case matching what you've provided in the parameters of your example.